tests/plots/test_summary.py

```diff
@@ -121,3 +122,16 @@ def test_random_summary_with_log_scale():
     shap.summary_plot(np.random.randn(20, 5), use_log_scale=True, show=False)
     fig.set_layout_engine("tight")
     return fig
+
+
+def test_no_type_error_in_summary_plot():
```

Comment:
Suggest adding a quick comment to link this test to the original issue:

```suggestion
def test_summary_plot_with_multiclass_model():
    # See GH #2893
```